Source/WebCore/loader/cache/CachedFontLoadRequest.h

```diff
@@ -39,8 +40,8 @@ class FontCreationContext;
 class CachedFontLoadRequest final : public FontLoadRequest, public CachedFontClient {
     WTF_MAKE_FAST_ALLOCATED_WITH_HEAP_IDENTIFIER(Loader);
 public:
-    CachedFontLoadRequest(CachedFont& font)
-        : m_font(&font)
+    CachedFontLoadRequest(CachedFont& font, Document* document)
+        : m_font(&font), m_document(document)
```

Comment:
`, m_document(document)` needs to be on the next line.

Reply:
I did 

`,  m_context(context)` on the next line.